src/content/en/2020/pwa.md

```diff
@@ -4,21 +4,437 @@ part_number: II
 chapter_number: 14
 title: PWA
 description: PWA chapter of the 2020 Web Almanac covering service workers (registations, installability, events and filesizes), Web App Manifests properties, and Workbox.
-authors: [hemanth, logicalphase]
-reviewers: [thepassle, jadjoubran, pearlbea, gokulkrishh, jaisanth]
+authors: [hemanth]
+reviewers: [thepassle, jadjoubran, pearlbea, gokulkrishh]
 analysts: [bazzadp]
 translators: []
-#hemanth_bio: TODO
-#logicalphase_bio: TODO
+hemanth_bio: <a href="https://h3manth.com">Hemanth HN</a> is a FOSS Computer polyglot, FOSS philospher, GDE for web and payments domain, DuckDuckGo community member, TC39 delegate and Google Launchpad Accelerator mentor. Loves The WEB && CLI. Hosts <a href="https://TC39er.us">TC39er.us</a> podcast.
 discuss: 2050
 results: https://docs.google.com/spreadsheets/d/1AOqCkb5EggnE8BngpxvxGj_fCfssT9sZ0ElCQKp4pOw/
 queries: 14_PWA
-#featured_quote: TODO
-#featured_stat_1: TODO
-#featured_stat_label_1: TODO
-#featured_stat_2: TODO
-#featured_stat_label_2: TODO
-#featured_stat_3: TODO
-#featured_stat_label_3: TODO
+featured_quote: Web Apps which progresses into a native like application is a PWA!
+featured_stat_1: 16.6%
+featured_stat_label_1: Percatage of PWA for the entire traffic.
+featured_stat_2: 27.97%
+featured_stat_label_2: Web apps that load fast enough for a PWA.
+featured_stat_3: 25%
+featured_stat_label_3: Percent of Mobile environments using importScripts.
 unedited: true
 ---
+
+## Introduction
+
+In 1990 we had the first ever browser called the “WorldWideWeb” and ever since the web and the browser has been evolving and for web to progress itself into a native behaviour is a big win special in this ear of mobile domination, URLs have provided an ubiquitous way to distribute information and nothing better than the browser to handle them and hence a technology which provides such capabilities to the browser becomes a game changer and Progressive Web Apps provide such advantages for the web to win yet again.
```

Comment:
```suggestion
In 1990 we had the first ever browser called the “WorldWideWeb” and ever since the web and the browser have been evolving and for the web to progress itself into a native behaviour is a big win special in this era of mobile domination, URLs have provided an ubiquitous way to distribute information and nothing better than the browser to handle them and hence a technology which provides such capabilities to the browser becomes a game changer and Progressive Web Apps provide such advantages for the web to win yet again.
```